packages/gestalt/src/Pulsar/Pulsar.css

```diff
@@ -1,86 +1,49 @@
 .innerCircle {
-  composes: absolute from "../Layout.css";
-  composes: circle from "../Borders.css";
-  composes: darkGrayBg from "../Colors.css";
-  animation-duration: 3s;
+  animation-delay: 0.8s;
+  animation-duration: 2.5s;
   animation-iteration-count: inherit;
-  animation-name: InnerAnimation;
+  animation-name: PulseAnimation;
+  border: 8px solid #0084ff;
+  border-radius: 50%;
   height: 100%;
+  opacity: 1;
   width: 100%;
 }
 
-.outerCircle {
-  composes: absolute from "../Layout.css";
-  composes: circle from "../Borders.css";
-  composes: darkGrayBg from "../Colors.css";
-  animation-duration: 3s;
-  animation-iteration-count: inherit;
-  animation-name: OuterAnimation;
-  height: 100%;
-  width: 100%;
+.innerCircle:hover {
```

Comment:
@dalvarez should we also add a `:focus` here with the same styles? How does this behave when you're navigating with the keyboard only